### ISSUE TITLE
Documentation and Output Formatting - salt.modules.win_status

### ISSUE THREAD

**Issue by sumeetisp** [saltstack/salt#37984]
### Description of Issue/Question
1)salt.modules.win_status.uptime
As per documentation,
salt.modules.win_status.uptime
Return the system uptime for this machine in seconds
human_readable
: False
If True, then return uptime in years, days, and seconds
I tried using the human_readable=true and it returned me uptime in hours,minutes and seconds.
Not in years, days, and seconds. So i guess the documentation should say Years,Days,Hours,Minutes and Seconds.

Secondly,

The output thrown out on the CLI is as follows,

```
salt minion2 status.uptime human_readable=true
minion2:
    12:53:06.891000
```

Which doesnt necessarily point that this is uptime. If i am running this with other cmds(may be via state files) which will show a bunch of output then finding uptime in that bunch would be difficult.

I expect an output which is as follows,

```
minion2:
  - Uptime:
     12:53:06.891000
```

2)salt.modules.win_status.cpuload
The output of the above module is as follows,

```
salt minion2 status.cpuload
minion2:
    0
```

Again the format of the output is not proper and this cpuload which is 0 is wrong. Same comments as for the 1st one.

### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)
Salt Version:
           Salt: 2016.3.4

Dependency Versions:
           cffi: 1.6.0
       cherrypy: 3.2.3
       dateutil: 1.5
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.21.1
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.5 (default, Aug  2 2016, 04:20:16)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: redhat 7.3 Maipo
        machine: x86_64
        release: 3.10.0-514.el7.x86_64
         system: Linux
        version: Red Hat Enterprise Linux Server 7.3 Maipo

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)

**Comment by Ch3LL** [comment]
@sumeetisp agreed we need to get the docs fixed up. 

ping @twangboy do you think we could add that extra output to the returns of those commands?

**Comment by twangboy** [comment]
@Ch3LL What does the output in Linux look like. We need to maintain parity in output formats where possible.

**Comment by twangboy** [comment]
@Ch3LL We definitely need to fix the cpuload problem though.

**Comment by twangboy** [comment]
`cpuload` has been fixed in 2017.7. The docs have been updated with https://github.com/saltstack/salt/pull/49059.

As far as changing the output of those commands, that's pretty standard across salt for modules that return a single value. I think it's unnecessary.
